### PR TITLE
fix package.json

### PR DESCRIPTION
**Pull request by marcgabe15**
# Description
resolves conflicting dependencies and hopefully fixes CI

# Related Issue

# Motivation and Context

# How Has This Been Tested?

# Screenshots (if appropriate):

### CODE REVIEW

**Comment by marcgabe15** [comment]
@kapooramanpreet  this should fix the package.json issues we are having for back end

File: .gitignore [hunk 1]
```diff
@@ -15,6 +15,7 @@ app.zip
 # react-scripts production
 /build
 .env
+environment_file
```
Comment (kapooramanpreet) [inline .gitignore:18]: What is stored in environment files generally Marc?

File: package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "@types/bcrypt": "^5.0.0",
     "@types/cors": "^2.8.12",
     "@types/express": "^4.17.13",
-    "@types/jest": "~26.0.23",
+    "@types/jest": "^27.0.0",
```
Comment (kapooramanpreet) [inline package.json:10]: Is there an automatic tool to identify major version changes? or did you manually have to lookup and change them? For future reference.

File: src/api/routes/v1/submission.routes.ts [hunk 3]
```diff
@@ -1,14 +0,0 @@
-import * as express from 'express';
-import * as submissionsController from '../../controllers/submission.controller';
-import multer from '../../middlewares/multer';
-import { authenticateJWT } from '../../middlewares/auth';
-const submissionRouter = express.Router();
-
-submissionRouter
-  .route('/upload')
```
Comment (kapooramanpreet) [inline src/api/routes/v1/submission.routes.ts:8]: Why did we have this route initially?

**Review by kapooramanpreet — APPROVED** [review]
Why are we deleting the submission/upload route and controller? Rest looks good to me.

File: package.json (hunk 2 above)
Comment (marcgabe15) [reply package.json:10]: in npm audit the peer dependencies tell you what the min version needs to be then manually backtrack from there

File: src/api/routes/v1/submission.routes.ts (hunk 3 above)
Comment (marcgabe15) [reply src/api/routes/v1/submission.routes.ts:8]: This was the automatic grader (i.e upload student submission and automatically grade them)

File: package.json (hunk 2 above)
Comment (kapooramanpreet) [reply package.json:10]: Thanks.

File: src/api/routes/v1/submission.routes.ts (hunk 3 above)
Comment (kapooramanpreet) [reply src/api/routes/v1/submission.routes.ts:8]: Thanks.

**Pull request merged**